src/jobs/publish-docs.yml

```diff
@@ -0,0 +1,53 @@
+executor: entur-cci-toolbox
+
+parameters:
+  gcp-service-key:
+    description: |
+      The credentials for the Service Account to be used when communicating to Google Cloud.
+      If not provided, the command will look for the environment variable $DOCS_PASSWORD
+
+      To obtain the key for this command,
+      use either the Google Cloud Platform Console or gcloud iam service-accounts keys create.
+      The key can be .json (preferred) or .p12 (legacy) format.
+    type: string
+    default: ""
+  project-name:
+    description: |
+      Specify the project name (e.g. mobility, customers). This is the same as the bucket name in GCS.
```

Comment:
Same as bucket name? Det er vel egentlig berre 1 bucket, med forskjellige undermapper. Kanskje heller skrive `This will be the name of this project's directory in the documentation GCS bucket.`